frontend/app/routes/grant.$type.tsx

```diff
@@ -1,11 +1,14 @@
-import { LoaderFunctionArgs } from '@remix-run/node'
+import { LoaderFunctionArgs } from '@remix-run/cloudflare'
 import { redirect } from '@remix-run/react'
 import { commitSession, getSession } from '~/lib/server/session.server'
 import { isGrantValidAndAccepted } from '~/lib/server/open-payments.server'
 
-export async function loader({ params, request }: LoaderFunctionArgs) {
-  const elementType = params.type
+export async function loader({ params, request, context }: LoaderFunctionArgs) {
+  const {
+    cloudflare: { env }
+  } = context
```

Comment:
Nit: Sometimes this is also viable:
```suggestion
  const { env } = context.cloudflare
```